gzip.go

```diff
@@ -162,9 +181,18 @@ func MustNewGzipLevelHandler(level int) func(http.Handler) http.Handler {
 // if an invalid gzip compression level is given, so if one can ensure the level
 // is valid, the returned error can be safely ignored.
 func NewGzipLevelHandler(level int) (func(http.Handler) http.Handler, error) {
+	return NewGzipLevelAndMinSize(level, DefaultMinSize)
+}
+
+// NewGzipLevelAndMinSize do as above but let caller specify the minimum size
```

Comment:
Can you specify "NewGzipLevelHandler" instead of "do as above"?

Also please end the doc comment with a period.